components/event-sources/apis/sources/v1alpha1/http_lifecycle.go

```diff
@@ -76,6 +82,16 @@ func (s *HTTPSourceStatus) MarkSink(uri string) {
 	httpCondSet.Manage(s).MarkTrue(HTTPConditionSinkProvided)
 }
 
+// MarkMonitoring sets the PolicyCreated condition to True once a Policy is created.
+func (s *HTTPSourceStatus) MarkMonitoring(policy *authenticationv1alpha1.Policy) {
```

Comment:
"MarkMonitoring" doesn't convey the intention. Did you mean "MarkMonitoringReady"?

Reply:
Changed to `MarkMonitoringReady`